gulpfile.js

```diff
@@ -14,6 +14,7 @@ var runSequence = require('run-sequence');
 var zip = require('gulp-zip');
 var mocha = require('gulp-mocha');
 var path = require('path');
+var walk = require('walk');
```

Comment:
Definitely. It's a generic tool for walking directory trees and could be used for just about anything.